tests/config-tests.js

```diff
@@ -0,0 +1,47 @@
+const expect = require('chai').expect
+const schema = require('./schema.js')
+const Ajv = require('ajv').default
+const ajv = new Ajv()
+const fs = require('fs')
+
+const platforms = [
```

Comment:
Can this be made a shared import with index.js?

That way if we add a new platform we don't miss off running tests against it. Either that or make a helper that generates the array based on the files it finds.